src/js/components/awardv2/idv/IdvContent.jsx

```diff
@@ -20,6 +20,7 @@ import { AWARD_V2_OVERVIEW_PROPS } from '../../../propTypes';
 
 const propTypes = {
     awardId: PropTypes.string,
+    counts: PropTypes.object,
     overview: AWARD_V2_OVERVIEW_PROPS,
```

Comment:
We could put a TODO here to add prop types for this object, or just define it here with `shape()` or put it in the `/prop-types` dir

Reply:
Added to `/prop-types` dir